portfolio/src/main/java/com/google/sps/servlets/QuoteGenerator.java

```diff
@@ -9,10 +12,38 @@
 /** Handles requests sent to the /quote URL endpoint */
 @WebServlet("/quote")
 public class QuoteGenerator extends HttpServlet {
+  /**
+   * Build the quotes using the author name and quote text.
+   *
+   * @param authorName The author of the quote's name
+   * @param quoteText  The quote text
+   * @return A map with the author's name and the text
+   */
+  private HashMap<String, String> buildQuote(final String authorName, final String quoteText) {
```

Comment:
Move private methods to be after all the public methods. So move this after doGet.

Reply:
Moved private methods to be after public methods